### DISCUSSION TITLE
Image rotation  90° and 270°

### DISCUSSION THREAD

**Discussion by BazzeDe** [roleoroleo/yi-hack-Allwinner-v2#681] [Ideas]
It would be great if the image could be rotated not only 180°, but also 90° and 270°. My cameras are hanging in portrait format. I think that would be a feature that would be very useful not only for me.

Sebastian

**Comment by roleoroleo** [comment]
Sorry but it's not possible.
The codec is managed by yi firmware.
And I can't decode and re-encode the stream because the cam doesn't have enough resources.

**Comment by BazzeDe** [comment]
That's a pity. Many thanks for the quick response.
